src/libs/Providers/LangChain.Providers.Google/LangChain.Providers.Google.csproj

```diff
@@ -0,0 +1,25 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+    <PropertyGroup>
+        <TargetFrameworks>net4.6.2;netstandard2.0;net6.0;net7.0;net8.0</TargetFrameworks>
+        <NoWarn>$(NoWarn);CA1305;CA1303;CA1835;CA1510;CA2234;CA2016;CA1849;CA1512</NoWarn>
+    </PropertyGroup>
+
+    <ItemGroup>
+      <PackageReference Include="Google_GenerativeAI" />
+    </ItemGroup>
+
+    <ItemGroup Label="Usings">
+        <Using Include="System.Net.Http" />
+    </ItemGroup>
+
+    <PropertyGroup Label="NuGet">
+        <Description>HuggingFace API LLM and Chat model provider.</Description>
```

Comment:
Please change description and tags

Reply:
Sorry about that, I changed the description